adafruit_emc2101/__init__.py

```diff
@@ -261,17 +281,23 @@ def external_temperature(self):
         temp_msb = self._ext_temp_msb
         full_tmp = (temp_msb << 8) | temp_lsb
         full_tmp >>= 5
-        full_tmp *= 0.125
-
-        return full_tmp
+        if full_tmp == emc2101_regs.TEMP_FAULT_OPENCIRCUIT or full_tmp == emc2101_regs.TEMP_FAULT_SHORT:
```

Comment:
Looks like `pylint` wants to rewrite this line as such:

```suggestion
        if full_tmp in (emc2101_regs.TEMP_FAULT_OPENCIRCUIT, emc2101_regs.TEMP_FAULT_SHORT):
```

`black` may want to reformat that after the fact.

Reply:
There are some times I feel these tools are too strict!

Reply:
It's actually a pretty neat trick, Pythonic and easy to read :) Definitely helpful when checking 3+ but still helpful for 2.

Reply:
I committed this fix, and then redid it when introducing exceptions because the two values now have an exception each.